### PR TITLE
Task #209835 [WEB] In camp->update learner->list of learner-> Give button to view profile of learner

### PR DESCRIPTION
**Pull request by chaitanyakole**
### I have ensured that following `Pull Request Checklist` is taken care of before sending this PR
* [x] Code is formatted as per format decided
* [x] Updated acceptance criteria in tracker
* [NA] Updated test cases in test-cases-tracker
https://tracker.tekdi.net/issues/209835

https://github.com/tekdi/eg-website/assets/124124878/b2908b9d-e0e9-4d3a-951f-58dc63f5ae26

### CODE REVIEW

File: apps/front-end/src/pages/front-end/Camp/CampSelectedLearners.js [hunk 1]
```diff
@@ -60,7 +82,10 @@ export default function CampSelectedLearners({
         edit_page_type: "edit_learners",
         id: camp_id?.id,
       };
+
+      console.log(updateLearner.learner_ids, "updateLearner");
```
Comment (swapnilphalke03) [inline apps/front-end/src/pages/front-end/Camp/CampSelectedLearners.js:86]: Remove Console.log

File: apps/front-end/src/pages/front-end/Camp/CampSelectedLearners.js [hunk 2]
```diff
@@ -60,7 +82,10 @@ export default function CampSelectedLearners({
...
       const data = await campService.updateCampDetails(updateLearner);
+      console.log("updatelearner", data);
```
Comment (swapnilphalke03) [inline apps/front-end/src/pages/front-end/Camp/CampSelectedLearners.js:88]: Remove Console.log

File: apps/front-end/src/pages/front-end/Camp/CampSelectedLearners.js [hunk 3]
```diff
@@ -195,6 +232,76 @@ export default function CampSelectedLearners({
           {t("SAVE_AND_CAMP_PROFILE")}
         </FrontEndTypo.Primarybutton>
       </Box>
+      <Modal isOpen={modalVisible} avoidKeyboard size="xl">
+        <Modal.Content>
+          <Modal.Header textAlign={"Center"}>{t("PROFILE")}</Modal.Header>
+          <Modal.Body>
+            {console.log(modalVisible)}
```
Comment (swapnilphalke03) [inline apps/front-end/src/pages/front-end/Camp/CampSelectedLearners.js:239]: Remove Console.log

**Review by swapnilphalke03 — CHANGES_REQUESTED** [review]
Remove Console.log

**Review by swapnilphalke03 — APPROVED** [review]

File: apps/front-end/src/pages/front-end/Camp/CampSelectedLearners.js [hunk 4]
```diff
@@ -124,68 +147,80 @@ export default function CampSelectedLearners({
...
+                        {item?.program_beneficiaries[0]?.enrollment_first_name}
+                        {item?.program_beneficiaries[0]
+                          ?.enrollment_middle_name &&
+                          ` ${item?.program_beneficiaries[0]?.enrollment_middle_name}`}
+                        {item?.program_beneficiaries[0]?.enrollment_last_name &&
+                          ` ${item?.program_beneficiaries[0]?.enrollment_last_name}`}
+                      </FrontEndTypo.H3>
+                      <Text>{item?.district}</Text>
+                      <Text>{item?.block}</Text>
+                      <Text>{item?.village}</Text>
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/Camp/CampSelectedLearners.js:209]: please use typo

**Review by sagarkoshti1990 — CHANGES_REQUESTED** [review]

**Pull request merged**